dts/bindings/serial/st,stm32-usart.yaml

```diff
@@ -29,4 +29,10 @@ properties:
       category: optional
       description: Set to enable RTS/CTS flow control at boot time
       generation: define
+
+    dma-channels:
```

Comment:
dma-channels is now part of base.yaml, this could be removed.

Reply:
Thanks for the hints, @erwango ! I was a bit confused by the following configuration in the dtsi files:
```
                        dmas = <&dma2 3 3 0x400 0x3
                                &dma2 2 3 0x400 0x3>
```

I know the second parameter is the stream no, the third is the channel no. What does the last two parameters mean? Where can I get their descriptions? I have thought `0x400` is the range of the interface registers (got the hint from I2S). However, from the descriptions of the file `vi st,stm32-dma.yaml`, it seems the 4th is `channel-config` and the last one is `features`, right? If so, I guess the configs on I2S could be wrong.

Reply:
Parameters description is provided in the dma bindings : `dts/bindings/dma/st,stm32-dma.yaml`

Reply:
I let @FRASTM comment on the I2S configuration

Reply:
Never mind. I guess the parameter 0x400 is correct option, which is defined as `increment address between transfers`.

Reply:
For stm32f4 DMA is V1. First param is the stream (from 0 to 7), second param is the request (aka channels) from 0 to 7 per DMA stream. Example for F411 (according to the ref Man), only DMA1 has requests for I2s (I2S2 or I2S3) 
I2S**2**_EXT_RX  on stream 3 channel request 3     I2S**3**_EXT_RX  on stream 2 channel request 2 
I2S**2**_EXT_TX  on stream 4 channel request 2     I2S**3**_EXT_TX  on stream 5 channel request 2